tests/unit/test_app.py

```diff
@@ -46,7 +42,7 @@ def get_request_body(cls, without=None, placements=None, update=None):
     @patch('provider.geo_provider.GeolocationProvider.__init__', return_value=None)
     def test_app_pulse(self, mock_geo):
         resp = self.create_client_no_geo_locs().get('/pulse')
-        self.assertEqual(resp.json, {"pulse" : "ok"})
+        self.assertEqual(resp.json(), {"pulse" : "ok"})
```

Comment:
Note that expect and actual are flipped in all these asserts. Will do another pr pass to flip these later.

Reply:
👍